core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -166,19 +158,11 @@ type DbEthTx struct {
 }
 
 func DbEthTxFromEthTx(ethTx *EvmTx) DbEthTx {
-	var from common.Address
-	var to common.Address
-	if ethTx.FromAddress != nil {
-		from = ethTx.FromAddress.Address
-	}
-	if ethTx.ToAddress != nil {
-		to = ethTx.ToAddress.Address
-	}
 	return DbEthTx{
 		ID:                 ethTx.ID,
 		Nonce:              ethTx.Nonce,
-		FromAddress:        from,
-		ToAddress:          to,
+		FromAddress:        ethTx.FromAddress.Address,
+		ToAddress:          ethTx.ToAddress.Address,
```

Comment:
Plz ensure all tests pass with this change.
I recall I had to do this, as some tests create an ethTx, with the ToAddress being nil, causing this line to crash.

Reply:
yup! everything is passing - it can't create a nil address because it's no longer a pointer